db/compaction/compaction_iterator.cc

```diff
@@ -617,6 +628,11 @@ void CompactionIterator::NextFromInput() {
             TEST_SYNC_POINT_CALLBACK(
                 "CompactionIterator::NextFromInput:KeepSDForWW",
                 /*arg=*/nullptr);
+          } else {
+            // We cannot drop the SingleDelete as timestamp is enabled, and
+            // timestamp of this key is less than *full_history_ts_low_. We will
```

Comment:
TODO: update the comment: key is greater than or equal to *full_history_ts_low_.